src/commands/utils/info.ts

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+/*
+    Animal Rights Advocates Discord Bot
+    Copyright (C) 2022  Anthony Berg, Kate Fort
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Command } from '@sapphire/framework';
+
+class InfoCommand extends Command {
+  argMap: Map<string, string>;
+
+  public constructor(context: Command.Context, options: Command.Options) {
+    super(context, {
+      ...options,
+      name: 'info',
+      description: 'Gets info on common anti-vegan arguments',
+    });
+  }
+
+  // Registers that this is a slash command
+  public override registerApplicationCommands(registry: Command.Registry) {
+    registry.registerChatInputCommand((builder) => builder
+      .setName(this.name)
+      .setDescription(this.description))
+      .addStringOption((option) => option.setName('argument')
+        .setDescription('The argument you want to learn about')
+        .setRequired(true));
```

Comment:
I think it may have been better to use [choices](https://discordjs.guide/interactions/slash-commands.html#choices) when selecting what anti-vegan argument they'd want